layouts/_default/baseof.html

```diff
@@ -1042,7 +1041,7 @@ <h4>Legal</h4>
     <script src="/assets/js/owl.carousel.min.js"></script>
     <script src="/assets/js/headroom.min.js"></script>
     <!-- masonary js  -->
-    <script src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
+    <!-- <script src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script> -->
```

Comment:
If this script is not necessary, then remove this line instead of keeping it commented out.